README.md

```diff
@@ -1 +1,236 @@
-# LeaderBoard
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="./murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+<h3><b>LeaderBoard</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📺 Presentation](#presentation)
+- [📝 License](#license) -->
+
+PROJECT DESCRIPTION
+
+# 📖 LeaderBoard using webPack <a name="about-project"></a>
+
+**LeaderBoard** In this project, we made a basic to do list due to given requirments that we used webPack for building an HTML template and to add CSS style automatically without link in HTML head.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>webPack</li>
+    <li>HTML</li>
+    <li>JavaScript</li>
+    <li>CSS</li>
+    </ul>
+</details>
+
+<!-- Features -->
+
+<!-- ### Key Features <a name="key-features"></a>
+
+- **Responsive design for both Desktop and Mobile**
+- **Speaker section geerated dynamically via js**
+- **Designed with mobile first concept** -->
+
+<!-- <p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/Fahim-Khan-P/LeaderBoard.git)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+ GETTING STARTED
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+```sh
+  git clone https://github.com/Fahim-Khan-P/LeaderBoard.git
+```
+
```

Comment:
- [ ] The Getting Started section could be more descriptive, kindly add the `Usage`, `Install` and `Setup` segments with the correct information for these segments